pom.xml

```diff
@@ -251,6 +253,16 @@
                 <artifactId>hadoop-client</artifactId>
                 <version>${hadoop.version}</version>
             </dependency>
+            <dependency>
+                <groupId>org.apache.hive</groupId>
+                <artifactId>hive-serde</artifactId>
```

Comment:
these dependencies are duplicated with hive/pom.xml.
In IoTDB, the new dependency need to be added in the main pom.xml file, and declare which one we  need to use in child pom.xml.

Reply:
Sorry, I can't get what you mean.
The dependencies of the main pom.xml file are declared between tag <dependencyManagement>, so I have to give the groupId and artifactId in child pom.xml in order to use them.